src/wp-includes/theme.php

```diff
@@ -3457,3 +3601,371 @@ function _wp_keep_alive_customize_changeset_dependent_auto_drafts( $new_status,
 		clean_post_cache( $post_id );
 	}
 }
+
+/**
+ * Creates the initial theme features when the 'setup_theme' action is fired.
+ *
+ * See {@see 'setup_theme'}
+ *
+ * @since 5.5.0
+ */
+function create_initial_theme_features() {
+	register_theme_feature(
+		'align-wide',
+		array(
+			'description'  => __( 'Whether theme opts in to wide alignment CSS class.' ),
+			'show_in_rest' => true,
+		)
+	);
+	register_theme_feature(
+		'automatic-feed-links',
+		array(
+			'description'  => __( 'Whether posts and comments RSS feed links are added to head.' ),
+			'show_in_rest' => true,
+		)
+	);
+	register_theme_feature(
+		'custom-header',
+		array(
+			'description'  => __( 'Custom header if defined by the theme.' ),
+			'type'         => 'object',
+			'show_in_rest' => array(
+				'schema' => array(
+					'properties' => array(
+						'default-image'      => array(
+							'type'   => 'string',
+							'format' => 'uri',
+						),
+						'random-default'     => array(
+							'type' => 'boolean',
+						),
+						'width'              => array(
+							'type' => 'integer',
+						),
+						'height'             => array(
+							'type' => 'integer',
+						),
+						'flex-height'        => array(
+							'type' => 'boolean',
+						),
+						'flex-width'         => array(
+							'type' => 'boolean',
+						),
+						'default-text-color' => array(
+							'type' => 'string',
+						),
+						'header-text'        => array(
+							'type' => 'boolean',
+						),
+						'uploads'            => array(
+							'type' => 'boolean',
+						),
+						'video'              => array(
+							'type' => 'boolean',
+						),
```

Comment:
I don't think so. I pulled those out when we landed [#49037](https://core.trac.wordpress.org/ticket/49037) since they can't always be safely exposed over the REST API since they aren't necessarily just a string function name. They could also be an anonymous function, or an object + method name array.